eclipse-collections-code-generator/src/main/resources/test/collection/mutable/abstractMutablePrimitiveCollectionTestCase.stg

```diff
@@ -398,23 +397,6 @@ public abstract class AbstractMutable<name>CollectionTestCase extends Abstract<n
         Assert.assertEquals(unmodifiableCollection, collection.asUnmodifiable());
     }
 
-    @Override
-    @Test(expected = NoSuchElementException.class)
-    public void <type>Iterator_throws_non_empty_collection()
-    {
-        super.<type>Iterator_throws_non_empty_collection();
-        Mutable<name>Collection collection = this.newWith();
-        collection.add(<(literal.(type))("1")>);
-        collection.add(<(literal.(type))("2")>);
-        collection.add(<(literal.(type))("3")>);
-        <name>Iterator iterator = collection.<type>Iterator();
-        while (iterator.hasNext())
-        {
-            iterator.next();
-        }
-        iterator.next();
-    }
-
```

Comment:
this test is almost identical to the superclass, but it uses add() instead of a varargs creation method. Calling add() is the reason it doesn't work in unmodifiable tests which results in even more overrides.